api/services/auth.js

```diff
@@ -1,9 +1,14 @@
 const nodemailer = require('nodemailer');
 const { hashHelpers, jwtHelpers, emailHelpers } = require('../helpers');
 const { ErrorHandler } = require('../middlewares/errorHandler');
+const ResetPasswordApproveError = require('../services/errorHandlers/resetPasswordApproveError');
+const ResetPasswordError = require('../services/errorHandlers/resetPasswordError');
+
```

Comment:
Why your import this 2 errors from different files? We have index.js in our error folder, so you can import from there

Reply:
Ok. Done